src/main/java/com/google/codeu/data/Post.java

```diff
@@ -0,0 +1,138 @@
+package com.google.codeu.data;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This class creates a Post object that is used for our User's posts and to
+ * populate the webpage.
+ * 
+ * @author Elli Kaplan
+ *
+ */
+public class Post {
+
+	private UUID postID;
+	private String creator;
+	private long timestamp;
+	private String coverInamgeUrl;
+	private LocationMarker marker;
+	private List<ChartDataRow> chartData;
+	private List<Message> mesages;
+
+	public Post(String creator, String coverImageUrl, LocationMarker marker, List<ChartDataRow> chartData,

```

Comment:
This Post constructor shouldnt use `List<Message> messages` mainly because when a post is created it will have no messages.

Also currently this is missing `title` and `content` you can see what those fields are in the drawings i put in the project design doc.

(i realize my PR does not have these fields, i missed them.)